ui-tests-starter/tst-243+/software/aws/toolkits/jetbrains/uitests/chatTests/qTestGeneration/QTestGenerationChatTestScripts.kt

```diff
@@ -0,0 +1,199 @@
+                    const elements = document.querySelectorAll('*');
+                    return Array.from(elements).find(element =>
+                        element.textContent?.trim() === expectedText
+                    );
+                },
+                {},
+                text
+            );
+            return element;
+        }
+        testNavigation().catch(console.error);
+""".trimIndent()
+
+val testNoFilePathScript = """
+const puppeteer = require('puppeteer');
+async function testNavigation() {
+    const browser = await puppeteer.connect({
+        browserURL: "http://localhost:9222"
+    })
+
+    try {
+
+        const pages = await browser.pages()
+        //console.log(pages)
+        for(const page of pages) {
+            const contents = await page.evaluate(el => el.innerHTML, await page.${'$'}(':root'));
+            //console.log(contents)
+            const element = await page.${'$'}('.mynah-chat-prompt-input')
+            if(element) {
+
+
+                const elements = await page.${'$'}${'$'}('.mynah-chat-command-selector-command');
+                const attr = await Promise.all(
+                    elements.map(elem => elem.evaluate(el => el.getAttribute('command')))
+                );
+                await page.type('.mynah-chat-prompt-input', '/test')
+
+                await page.keyboard.press('Enter');
+                await page.keyboard.press('Enter');
+
+                try {
+                    await waitForElementWithText(page, "Q - Test")
+                    console.log("new tab opened")
+
+                    const errorMessage = await page.waitForSelector('text/Sorry, there isn\'t a source file open right now that I can generate a test for. Make sure you open a source file so I can generate tests.', {
+                        timeout: 5000
+                    })
+                    console.log('Error message:', await errorMessage.evaluate(el => el.textContent))
+                } catch (e) {
+                    console.log("Element with text not found")
+                    console.log(e)
+                }
+
+            }
+        }
+
+
+    } finally {
+        await browser.close();
+    }
+}
+
+async function waitForElementWithText(page, text) {
+    await page.waitForFunction(
+        (expectedText) => {
+            const elements = document.querySelectorAll('*');
+            return Array.from(elements).find(element =>
+                element.textContent?.trim() === expectedText
+            );
+        },
+        {},
+        text
+    );
+}
+
+async function waitAndGetElementByText(page, text) {
+    const element = await page.waitForFunction(
+        (expectedText) => {
+            const elements = document.querySelectorAll('*');
+            return Array.from(elements).find(element =>
+                element.textContent?.trim() === expectedText
+            );
+        },
+        {},
+        text
+    );
+    return element;
+}
+
```

Comment:
nit: can you please delete extra lines? Not sure if the linter is working for this one